### PR TITLE
sync_diff_inspector: support bit type && add readme

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
1. we can use sql like `select i+0 from test.test` to  get bit type column's value with decimalism 
2. add readme for sync_diff_inspector

### CODE REVIEW

File: sync_diff_inspector/README.md [hunk 1]
```diff
@@ -1 +1,30 @@
 # sync-diff-inspector
+
+sync-diff-inspector is a tool for diff two database's data.
```
Comment (IANTHEREAL) [inline sync_diff_inspector/README.md]: s/diff/compare

File: sync_diff_inspector/README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+        the percent of sampling check (default 100)
+  -source-snapshot string
+        source database's snapshot config
+  -target-snapshot string
+        target database's snapshot config
+  -use-rowid
+        set true if target-db and source-db all support tidb implicit column _tidb_rowid
+```
+
+For more detail you can read the config.toml.
```
Comment (IANTHEREAL) [inline sync_diff_inspector/README.md]: `For more details`

**Comment by IANTHEREAL** [comment]
any jira issue or problem description?

**Comment by WangXiangUSTC** [comment]
jira: [`https://internal.pingcap.net/jira/browse/TOOL-207`](https://internal.pingcap.net/jira/browse/TOOL-207)  @GregoryIan

**Comment by IANTHEREAL** [comment]
I would confirm and fix it, close this pr now

**Pull request closed without merging**